base/include/MotionVectorExtractor.h

```diff
@@ -24,6 +24,7 @@ class MotionVectorExtractorProps : public ModuleProps
 		return ModuleProps::getSerializeSize() + sizeof(sendDecodedFrame) + sizeof(motionVectorThreshold);
 	}
 	bool sendDecodedFrame = false;
+	bool sendOverlayFrame = false;
```

Comment:
should be added in serialization part below

Reply:
Done